easybuild/easyconfigs/s/SIMPLE/SIMPLE-3.0.0-foss-2020b.eb

```diff
@@ -0,0 +1,80 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/03
+easyblock = 'CMakeMake'
+
+name = 'SIMPLE'
+version = '3.0.0'
+
+homepage = 'https://simplecryoem.com/'
+description = """Single-particle IMage Processing Linux Engine SIMPLE is an open-source software
+package for analysis of cryogenic transmission electron microscopy (cryo-EM) 
+movies of single-particles (Single-Particle Analysis, SPA)."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'opt': False}  # simple_test_units fails with opt=True
+
+source_urls = [GITHUB_SOURCE]
+github_account = 'hael'
+sources = ["v%(version)s.tar.gz"]
+checksums = ['9cf98d042b918170e3a9b92148c7081d35e0ef1c9fa8b6ea16cf2bd476945dd7']
+
+builddependencies = [('CMake', '3.18.4')]
+dependencies = [
+    ('XZ', '5.2.5'),
+    ('LibTIFF', '4.1.0'),
+    ('jbigkit', '2.1'), ]
+
+# Avoid using problematic FindOpenMP_Fortran.cmake; 
+# Instead OpenMP_Fortran_FLAGS will be set in configopts.
+preconfigopts = 'sed -i "s/unset (OpenMP_Fortran_FLAGS CACHE)/#&/g" ../*/cmake/Modules/ZSetParallelLibrary.cmake &&'
+
+local_configopts = " -DFFTW_DIR=$EBROOTFFTW "
+local_configopts += "-DTIFF_INCLUDE_DIR=$EBROOTLIBTIFF/include "
+local_configopts += "-DJBIG_LIBRARY=$EBROOTJBIGKIT/lib/libjbig.so "
```

Comment:
@ThomasHoffmann77 while it's a bit pedantic, could you apply `SHLIB_EXT` here too please, and in the line below?